src/Revise.jl

```diff
@@ -319,7 +319,11 @@ function parse_source!(md::ModDict, file::AbstractString, mod::Module, path)
         warn("omitting ", file, " from revision tracking")
         return false
     end
-    parse_source!(md, readstring(file), Symbol(file), 1, mod, path)
+    if VERSION >= v"0.7.0-DEV"
```

Comment:
I can't find the `read` method in the latest release of the `Compat` package 🤷‍. And `@macroexpand @compat read(x, String)` returns `:(read(x, String))` on both 0.6 and master. I found the commit, though, so will update the version check.